### DISCUSSION TITLE
i cant sign-in or sign-up :no_entry_sign:

### DISCUSSION THREAD

**Discussion by iamyahia** [vercel/community#4049] [Account Issue] [answered]
### Question

i try to sign-up with vercel with my github account 
when i click on github icon, it open a popup module and say to write my number .
when i type my phone number it gives this error: `An unexpected internal error occurred` . ( i'm from iraq  )
![verce](https://github.com/vercel/community/assets/90482598/5e6f587b-4a04-4418-ab22-83c176fc9682)

that's for a month  i try to sign-up but i cant 
and when i try other methods like email or password it gives me same popover module and same error.
please someone help me

**Comment by CynthiaWeaver-Jeannine** [comment]
I am having this problem as well. I signed up using github, entered a credit card and used the service one time. Now I cannot sign.--phone number is already associated with an account.  I can't get support on the site because you need to be signed in to contact support.  Does anyone have an idea for a solution to this issue, please?

**Comment by amyegan** [comment, marked as answer]
Sorry you're getting errors with phone verification. We can't safely discuss account information in public because of security concerns. 

Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will get you in contact with the team that can help with login issues.

**Comment by iamyahia** [comment]
i solve this problem by those steps :
 
 i'm linux user, so i try multiple times for a month with same error 
 but 
 suddenly we try in my friend laptop/windows and try to sign-up and create an account 
 so 
 then i try to sign-in with my linux so it's work succesfully
